common/url.go

```diff
@@ -438,6 +438,18 @@ func ParseServiceKey(serviceKey string) (string, string, string) {
 	return serviceKey, group, version
 }
 
+// IsAnyCondition judges if is any condition
+func IsAnyCondition(subscribedKey string, subscribedURL, serviceURL *URL) bool {
```

Comment:
The parameters are too confused. IMO, it should be: `func IsAnyCondition(intf, group, version string) bool`.